resources/assets/tconstruct/book/zh_CN/intro/toolstation.json

```diff
@@ -0,0 +1,8 @@
+{
+  "title":"工具装配台,
```

Comment:
Your JSON is missing a quotation mark here, that will most likely break things in game